src/app/telegram/votigram/components/CommonDrawer/index.tsx

```diff
@@ -3,7 +3,8 @@ import { Drawer, DrawerProps } from 'antd';
 import Alloyfinger from 'alloyfinger';
 import './index.css';
 import { LeftArrowOutlined } from '@aelf-design/icons';
-import { CloseIcon } from 'components/Icons';
```

Comment:
I have moved this icon to the components/Icons directory because this icon is also used in other places.

Reply:
I had change to components